tests/metal/simple-task.slang

```diff
@@ -1,6 +1,11 @@
-//TEST:SIMPLE(filecheck=CHECK): -target metal
-//TEST:SIMPLE(filecheck=CHECK-ASM): -target metallib
-//TEST:REFLECTION(filecheck=REFLECT):-target metal -entry main_kernel -stage task
+// TEST:SIMPLE(filecheck=CHECK): -target metal
```

Comment:
There shouldn't be a space between `//` and `TEST`, otherwise the test won't get picked up by our test driver.